src/package_managers.rs

```diff
@@ -250,6 +250,27 @@ impl ManagerInfo {
 
         Some(result as u64)
     }
+
+    pub fn process_homebrew_packages_count(&self) -> Option<u64> {
```

Comment:
Move this fn into the packages module itself, as it's only counting, not actually fetching package info.  I'll also fix this for flatpak later.

Reply:
Are we aiming for std to be optional behind feature flags, or can I import read_dir on the top of the file?